stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -252,6 +252,18 @@ This property allows the user to add a custom `icon` at the right of the input a
   </Story>
 </Canvas>
 
+## Border
+
+The `withBorder` property allows the user to add a custom themed border to the TextField.
```

Comment:
I think this sentence implies that the user can have control (`custom`) over the color of the border when it is simply themed like the rest, don't you think?

Reply:
Yes you're absolutely right